packages/data-context/src/actions/WizardActions.ts

```diff
@@ -295,14 +308,16 @@ export class WizardActions {
     }
   }
 
-  private async scaffoldComponentIndexHtml (chosenFramework: WizardFrontendFramework): Promise<NexusGenObjects['ScaffoldedFile']> {
+  private async scaffoldComponentIndexHtml (chosenFramework: Cypress.ResolvedComponentFrameworkDefinition): Promise<NexusGenObjects['ScaffoldedFile']> {
     const componentIndexHtmlPath = path.join(this.projectRoot, 'cypress', 'support', 'component-index.html')
 
     await this.ensureDir('support')
 
+    const defaultComponentIndex = componentIndexHtmlGenerator('')
+
     return this.scaffoldFile(
       componentIndexHtmlPath,
-      chosenFramework.componentIndexHtml(),
+      chosenFramework.componentIndexHtml?.() ?? defaultComponentIndex(),
```

Comment:
The reason this is now nullable is since third parties CAN provide a custom `component-index.html`, but don't need to. If they don't, they get our default one.